resources/core/charts/helm-broker/charts/etcd/Chart.yaml

```diff
@@ -1,10 +1,3 @@
-# Version from not yet merged PR: https://github.com/kubernetes/charts/pull/4678
 name: etcd
-home: https://github.com/coreos/etcd
-version: 0.4.0
-appVersion: 3.2.14
 description: Distributed reliable key-value store for the most critical data of a distributed system.
-icon: https://raw.githubusercontent.com/coreos/etcd/master/logos/etcd-horizontal-color.png
-sources:
-- https://github.com/coreos/etcd
-
+version: 0.0.1
```

Comment:
could you try to use etcd from service catalog? I think we should reduce number of etcds to one for SC specifc components.

Reply:
I've tried to achieve this with the `requirements.yaml` but it's not easy and the base chart still needs to be copied into sub-chart. I didn't find any other way how to do that.